kinetic/distribution.yaml

```diff
@@ -11449,6 +11449,22 @@ repositories:
       url: https://github.com/roboception/rc_genicam_api.git
       version: master
     status: developed
+  rc_genicam_driver:
+    doc:
+      type: git
+      url: https://github.com/roboception/rc_genicam_driver_ros.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/roboception-gbp/rc_genicam_driver_ros-release.git
+      version: 0.2.0-1
+    source:
+      test_pull_requests: true
```

Comment:
@flixr Have you [configured your repository](http://wiki.ros.org/buildfarm/Pull%20request%20testing) accordingly?

Reply:
jep, the ros-pull-request-builder is a collaborator on the whole https://github.com/roboception-gbp group